controllers/snapshot/snapshot_adapter.go

```diff
@@ -158,7 +158,7 @@ func (a *Adapter) EnsureAllReleasesExist() (results.OperationResult, error) {
 		return results.ContinueProcessing()
 	}
 
-	if !gitops.IsSnapshotCreatedByPushEvent(a.snapshot) {
+	if gitops.IsSnapshotCreatedByPACPullRequestEvent(a.snapshot) {
 		a.logger.Info("The Snapshot won't be released because it's not created by a push event.")
```

Comment:
Maybe we can also change this message to say:
```
The Snapshot won't be released because it was created for a PaC pull request event.
```

Reply:
Thanks! Updated as you suggested.